src/validations/input/loginValidation.js

```diff
@@ -0,0 +1,18 @@
+const inputValidation = require('./inputValidation');
+
+class LoginValidation extends inputValidation {
+  constructor() {
+    super();
+    this.getLogger = this.getLogger.bind(this);
+    this.validateInput = this.validateInput.bind(this);
+  }
+
+  validateInput(username, password) {
```

Comment:
IMO validators shouldn't do only one thing: you can group several input validations in one validator.